src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/services/citizen/SettlementReachedCitizenNotificationServiceTest.java

```diff
@@ -0,0 +1,162 @@
+package uk.gov.hmcts.cmc.claimstore.controllers.services.citizen;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.cmc.claimstore.MockSpringTest;
+import uk.gov.hmcts.cmc.claimstore.events.offer.AgreementCounterSignedCitizenActionsHandler;
+import uk.gov.hmcts.cmc.claimstore.events.offer.AgreementCountersignedEvent;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.offers.MadeBy;
+import uk.gov.hmcts.cmc.domain.models.offers.Settlement;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleResponse;
+import uk.gov.hmcts.cmc.domain.models.sampledata.offers.SampleOffer;
+import uk.gov.service.notify.NotificationClientException;
+
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class SettlementReachedCitizenNotificationServiceTest extends MockSpringTest {
+
+    private static final byte[] PDF_CONTENT = {1, 2, 3, 4};
+
+    @Autowired
+    private AgreementCounterSignedCitizenActionsHandler handler;
+
+    @Captor
+    private ArgumentCaptor<String> senderArgument;
+
+    @Captor
+    private ArgumentCaptor<String> templateArgument;
+
+    @Captor
+    private ArgumentCaptor<String> referenceArgument;
+
+    @Captor
+    private ArgumentCaptor<Map<String, String>> emailDataArgument;
+
+    private Claim claim;
+    private AgreementCountersignedEvent event;
```

Comment:
I do not think we need that field. Makes sense to reimplement this as method local variable.

Reply:
removed